INSTALL.rst

```diff
@@ -0,0 +1,363 @@
+latest git version :ref:`install-from-git`.
+
+The standard environment variables `CC`, `CXX`, `PKG_CONFIG` are respected.
+This means you can set them if your toolchain is prefixed. This may be used for
+cross compiling.
+::
+
+  export CC=x86_64-pc-linux-gnu-gcc
+  export CXX=x86_64-pc-linux-gnu-g++
+  export PKG_CONFIG=x86_64-pc-linux-gnu-pkg-config
+
+Once you have satisfied the requirements detailed below (mainly
+python, numpy, libpng and freetype), you can build Matplotlib.
+::
+
+  cd matplotlib
+  python setup.py build
+  python setup.py install
+
+We provide a `setup.cfg
+<https://raw.githubusercontent.com/matplotlib/matplotlib/master/setup.cfg.template>`_
+file that goes with :file:`setup.py` which you can use to customize
+the build process. For example, which default backend to use, whether
+some of the optional libraries that Matplotlib ships with are
+installed, and so on.  This file will be particularly useful to those
+packaging Matplotlib.
+
+If you have installed prerequisites to nonstandard places and need to
+inform Matplotlib where they are, edit ``setupext.py`` and add the base
+dirs to the ``basedir`` dictionary entry for your ``sys.platform``.
+e.g., if the header to some required library is in
+``/some/path/include/someheader.h``, put ``/some/path`` in the
+``basedir`` list for your platform.
+
+.. _install_requirements:
+
+Dependencies
+------------
+
+Matplotlib requires a large number of dependencies:
+
+  * `Python <https://www.python.org/downloads/>`_ (>= 2.7 or >= 3.4)
+  * `numpy <http://www.numpy.org>`_ (>= |minimum_numpy_version|)
+  * `setuptools <https://setuptools.readthedocs.io/en/latest/>`__
+  * `dateutil` (>= 1.1)
+  * `pyparsing <https://pyparsing.wikispaces.com/>`__
+  * `libpng <http://www.libpng.org>`__ ( >= 1.2) 
+  * `pytz <http://pytz.sourceforge.net/>`__
+  * FreeType (>= 2.3)
+  * `cycler <http://matplotlib.org/cycler/>`__ (>= 0.10.0)
+  * `six <https://pypi.python.org/pypi/six>`_
+  * `backports.functools_lru_cache <https://pypi.python.org/pypi/backports.functools_lru_cache>`_
+    (for Python 2.7 only)
+  * `subprocess32 <https://pypi.python.org/pypi/subprocess32/>`_ (for Python
+    2.7 only, on Linux and mac os X only)
+
+Optionally, you can also install a number of packages to enables better user
+interface toolkit. See :ref:`what-is-a-backend` for more details on the
+optional Matplotlib backends and the capabilities they provide.
+
+  * :term:`tk` (>= 8.3, != 8.6.0 or 8.6.1): for the TkAgg backend;
+  * `PyQt4 <https://pypi.python.org/pypi/PyQt4>`_ (>= 4.4): for the Qt4Agg backend;
+  * `PyQt5 <https://pypi.python.org/pypi/PyQt5>`_: for the Qt5Agg backend;
+  * :term:`pygtk` (>= 2.4): for the GTK and the GTKAgg backend;
+  * :term:`wxpython` (>= 2.8 or later): for the WX or WXAgg backend;
+  * `pycairo <https://pypi.python.org/pypi/pycairo>`_;
+  * `PySide <https://pypi.python.org/pypi/PySide>`_;
+  * `PyGTK <https://pypi.python.org/pypi/PyGTK>`_;
```

Comment:
Duplicated.  Also check the "what is a backend table" for how this plays with PyGObject.